ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py

```diff
@@ -21,14 +21,14 @@
     native_array=helpers.array_bools(num_arrays=1),
 )
 def test_numpy_sinh(
-    dtype_and_x,
-    dtype,
-    where,
-    as_variable,
-    with_out,
-    num_positional_args,
-    native_array,
-    fw,
+        dtype_and_x,
```

Comment:
shouldn't make changes to other functions in this PR

Reply:
There are no lint errors with the function I worked on, other lint errors are linked to functions that were not written by me. Please I would need further correction and clarification if I am allowed to tamper with other functions. Thanks